homeassistant/components/sensor/__init__.py

```diff
@@ -77,5 +77,15 @@ async def async_unload_entry(hass, entry):
     return await hass.data[DOMAIN].async_unload_entry(entry)
 
 
-class SensorEntity(Entity):
+class SensorEntity(MeasurableUnitEntity):
```

Comment:
Should all sensors get this logic or should be applied to the right ones? I guess that is a migration we can't do anymore at this point.

Reply:
Right, because it's a migration it's not feasible to do anything else.
We could do something like this though:

```py
class SensorEntity(Entity):
    """Base class for sensor entities"""

class MeasurableSensorEntity(MeasurableUnitEntity, SensorEntity):
    """Base class for sensor entities which represent a measurable unit."""
"""

The migration would then be to `MeasurableSensorEntity`, but integrations can move to `SensorEntity` for sensors which don't represent a measurable unit?